flagscale/train/train.py

```diff
@@ -1141,7 +1134,7 @@ def training_log(loss_dict, total_loss_dict, learning_rate, decoupled_learning_r
         total_loss_dict[nan_iters_key] = 0
         print_rank_last(log_string)
         if not args.auto_tune:
-            if report_memory_flag and learning_rate > 0.:
```

Comment:
Why does this pr remove the `learning_rate > 0`?

Reply:
Cause Megatron-LM removed it, but we didn't in the previous merge-megatron pr.